src/main/resources/application.properties

```diff
@@ -303,6 +303,8 @@ classification-service.password=
 # Classification Job Timeout in minutes
 classification-service.job.abort-after-minutes=45
 
+# Queue containing status of classification
+classification-service.job.destination-response=classification-jobs-status
```

Comment:
This queue is where Snowstorm requests Classification Service to write a response to after processing a classification. I suspect this value will change per environment.

Reply:
We need to make a breaking change note in the release notes about this because people will now need a JMS broker set up to be able to use classification. Either that or we make this backwards compatible with a config option to switch between the two strategies. If the status update strategy is polling then `getStatusChange` would always fetch the latest status? The `ClassificationService` class would have to check if the status is null and if it's new or not..

Reply:
Great suggestion, thanks Kai. 

I have changed this by adding `JMSListenerClassificationStatusService` and `RestTemplateClassificationStatusService` which can be controlled via the property `classification-service.job.status.use-jms`.